lib/Target/ExportSMTLIB/ExportSMTLIB.cpp

```diff
@@ -635,6 +635,10 @@ static LogicalResult emit(SolverOp solver, const SMTEmissionOptions &options,
       return op->emitError()
              << "solver must not contain any non-SMT operations";
 
+    if (isa<smt::Int2BVOp>(op))
+      return op->emitError(
+          "int2bv operations are not supported for SMTLIB emission");
```

Comment:
Maybe move this error message to `visitUnhandledSMTOp` in ExpressionVisitor (I don't quite remember why we just return success there, returning failure would make more sense (and explicitly handle ops for which we just want to return success, if there are any, might also just be a copy-paste mistake from StatementVisitor).